platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/JettyAuthenticator.java

```diff
@@ -98,29 +95,63 @@ public void doFilter(
     if (!CollectionUtils.isEmpty(sortedSecurityFilterServiceReferences)) {
       LOGGER.debug(
           "Found {} filter(s), now filtering...", sortedSecurityFilterServiceReferences.size());
-      final ProxyFilterChain chain = new ProxyFilterChain(filterChain);
+      final ProxyFilterChain chain = new ProxyFilterChain();
 
       // Insert the SecurityFilters into the chain one at a time (from lowest service ranking
       // to highest service ranking). The SecurityFilter with the highest service-ranking will
-      // end up at index 0 in the FilterChain, which means that the SecurityFilters will be
+      // end up at index 0 in the FilterChain, whichHttpSessionInvalidator means that the
+      // SecurityFilters will be
       // run in order of highest to lowest service ranking.
       for (ServiceReference<SecurityFilter> securityFilterServiceReference :
           sortedSecurityFilterServiceReferences) {
         final SecurityFilter securityFilter =
             bundleContext.getService(securityFilterServiceReference);
 
         if (!hasBeenInitialized(securityFilterServiceReference, bundleContext)) {
```

Comment:
❓ I was not sure if this was still necessary or not even after looking at code not sure why the filter needs to be initialized.